### PR TITLE
feat (doc): add a new homepage

### PR DESCRIPTION
**Pull request by rayene**
Still need to check licensing/copyright stuff.

### CODE REVIEW

File: docs/layouts/partials/landscape.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+     {{/* NOTE: the Site's title, and if there is a page title, that is set too */}}
+    <title>{{ block "title" . }}{{ .Site.Title }} {{ with .Title }} | {{ . }}{{ end }}{{ end }}</title>
+    <meta name="HandheldFriendly" content="True">
+    <meta name="MobileOptimized" content="320">
+
+    <meta name="viewport" content="width=device-width,minimum-scale=1">
+     {{ .Hugo.Generator }}
+
+      {{ if eq (getenv "HUGO_ENV") "production" | or (eq .Site.Params.env "production")  }}
```
Comment (yesnault) [inline docs/layouts/partials/landscape.html]: HUGO_ENV to remove

File: docs/layouts/partials/landscape.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      {{ else }}
+        <META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW">
+      {{ end }}
+
+     <link href='{{ "css/homepage.css" | relURL }}' rel='stylesheet' type="text/css" />
+      {{- template "_internal/opengraph.html" . -}}
+      {{- template "_internal/google_news.html" . -}}
+      {{- template "_internal/schema.html" . -}}
+
+      {{ if eq (getenv "HUGO_ENV") "production" | or (eq .Site.Params.env "production")  }}
```
Comment (yesnault) [inline docs/layouts/partials/landscape.html]: HUGO_ENV to remove

File: docs/layouts/partials/landscape.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        {{ template "_internal/google_analytics_async.html" . }}
+      {{ end }}
+
+
+
+
+  </head>
+  <body class="ma0 sans-serif bg-primary-color-light{{ with getenv "HUGO_ENV" }} {{ . }}{{ end }}">
```
Comment (yesnault) [inline docs/layouts/partials/landscape.html]: HUGO_ENV to remove

File: docs/layouts/partials/landscape.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+     {{ .Hugo.Generator }}
+
+      {{ if eq (getenv "HUGO_ENV") "production" | or (eq .Site.Params.env "production")  }}
+        <META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
+      {{ else }}
+        <META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW">
+      {{ end }}
+
+     <link href='{{ "css/homepage.css" | relURL }}' rel='stylesheet' type="text/css" />
+      {{- template "_internal/opengraph.html" . -}}
```
Comment (yesnault) [inline docs/layouts/partials/landscape.html]: opengraph.html to remove

File: docs/layouts/partials/landscape.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+      {{- template "_internal/google_news.html" . -}}
```
Comment (yesnault) [inline docs/layouts/partials/landscape.html]: google_news.html to remove

File: docs/layouts/partials/landscape.html [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+      {{- template "_internal/schema.html" . -}}
```
Comment (yesnault) [inline docs/layouts/partials/landscape.html]: schema.html

File: docs/layouts/partials/home-page-sections/open-source-involvement.html [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+</div>
+
+<div class="flex-ns flex-wrap center pb4 center mw9">
+  <!-- LEFT -->
+  <div class="w-100 tc w-third-l">
+    <h3 class="f3 mv3 accent-color-light">We welcome all contributions</h3>
+    <ul class="list ma0 pa0">
+      <li class="mb3 f4">
+
+        <a href="https://github.com/ovh/cds/blob/master/CONTRIBUTING.md" class="link mid-gray dim" target="blank">
```
Comment (bnjjj) [inline docs/layouts/partials/home-page-sections/open-source-involvement.html]: _blank

File: docs/layouts/partials/home-page-sections/open-source-involvement.html [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+      <p class="f4">CDS is open-source and completely free.</p>
+    </div>
+  </div>
+
+
+  <div class="w-100 tc w-third-l">
+    <h3 class="f3 mv3 accent-color-light">More ways to contribute</h3>
+    <ul class="list ma0 pa0">
+      <li class="mb3 f4">
+        <a href="introduction/" class="link mid-gray dim" target="blank">
```
Comment (bnjjj) [inline docs/layouts/partials/home-page-sections/open-source-involvement.html]: _blank

File: docs/layouts/partials/landscape.html [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+      {{ if eq (getenv "HUGO_ENV") "production" | or (eq .Site.Params.env "production")  }}
+        {{ template "_internal/google_analytics_async.html" . }}
```
Comment (bnjjj) [inline docs/layouts/partials/landscape.html]: I think we can delete

File: docs/layouts/partials/site-footer.html [hunk 10]
```diff
@@ -0,0 +1,10 @@
+<footer class="bg-primary-color-dark ph4-ns pt4 relative w-100" role="contentinfo">
+      <div class="b f3 center w4 light-gray mb3 nested-links tc">
+          <a href="https://github.com/ovh/cds/graphs/contributors" class="link"><img src="images/cds-transparent.png"> Contributors</a><br />
+      </div>
+    <div class="f7 gray mb5 mb0-ns ph3 w-100"> 
+      <p class="dib mr4">The CDS logos are copyrighted.</p>
+      <p class="dib">Icons made by <a href="https://www.flaticon.com/authors/eucalyp"  class="dim link light-gray pv2" title="Eucalyp">Eucalyp</a> from <a class="dim link light-gray pv2" href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> licensed by <a class="dim link light-gray pv2" href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></p>
```
Comment (fsamin) [inline docs/layouts/partials/site-footer.html]: it seems it should be a link with the text "`designed by {Author's Name} from Flaticon"`
 regarding https://file000.flaticon.com/downloads/license/license.pdf

**Comment by fsamin** [comment]
`````html
<!-- Place this tag where you want the button to render. -->
<a class="github-button" href="https://github.com/ntkme/github-buttons" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star ntkme/github-buttons on GitHub">Star</a>

<!-- Place this tag in your head or just before your close body tag. -->
<script async defer src="https://buttons.github.io/buttons.js"></script>
`````

**Comment by rayene** [comment]
@fsamin fixed.

**Pull request merged**